src/app/pair_game/_components/steps/IntroStepDesktop.tsx

```
@@ -0,0 +1,135 @@
+            onClick={onShareLink}
+          >
+            <Button
+              type="button"
+              variant="secondary"
+              size="full"
+              className="bg-[#FFE7ED]"
+            >
+              <Body3 weight="bold" className="text-game-primary">
+                게임 링크 공유하기
+              </Body3>
+            </Button>
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)